frontend/public/components/search.tsx

```diff
@@ -0,0 +1,264 @@
+          <ChipGroup withToolbar defaultIsOpen={false}>
+            <ChipGroupToolbarItem key="resources-category" categoryName="Resource">
+              {selectedItems.map((chip) => (
+                <Chip key={chip} onClick={() => updateSelectedItems(chip)}>
+                  <ResourceIcon kind={chip} />
+                  {chip.split('~').pop()}
+                </Chip>
+              ))}
+              {selectedItems.length > 0 && (
+                <span>
+                  <Button variant="plain" aria-label="Close" onClick={() => clearSelectedItems()}>
+                    <CloseIcon />
+                  </Button>
+                </span>
+              )}
+            </ChipGroupToolbarItem>
+            <ChipGroupToolbarItem key="label-category" categoryName="Label">
+              {labelFilter.map((chip) => (
+                <Chip key={chip} onClick={() => removeLabelFilter(chip)}>
+                  {chip}
+                </Chip>
+              ))}
+              {labelFilter.length > 0 && (
+                <span>
+                  <Button variant="plain" aria-label="Close" onClick={() => clearLabelFilter()}>
+                    <CloseIcon />
+                  </Button>
+                </span>
+              )}
+            </ChipGroupToolbarItem>
+            <ChipGroupToolbarItem key="name-category" categoryName="Name">
+              {nameFilter.map((chip) => (
+                <Chip key={chip} onClick={() => removeNameFilter(chip)}>
+                  {chip}
+                </Chip>
+              ))}
+              {nameFilter.length > 0 && (
+                <span>
+                  <Button variant="plain" aria-label="Close" onClick={() => clearNameFilter()}>
+                    <CloseIcon />
+                  </Button>
+                </span>
+              )}
+            </ChipGroupToolbarItem>
+          </ChipGroup>
+          {(selectedItems.length > 0 || labelFilter.length > 0 || nameFilter.length > 0) && (
+            <Button variant="link" key="clear-filters" onClick={() => clearAll()}>
+              Clear all filters
+            </Button>
+          )}
+        </div>
+      </PageHeading>
+      <div className="co-search co-m-pane__body">
+        {selectedItems.map((item) => {
+          return (
+            <Expandable
+              key={item}
+              toggleText={getToggleText(item)}
+              onToggle={() => toggleKindExpanded(item)}
+              isExpanded={selectedItems.length === 1 ? true : isKindExpanded(item)}
+            >
+              <ResourceList
+                kind={item}
+                selector={selector}
+                nameFilter={nameFilter.join(',')}
+                namespace={namespace}
+                mock={noProjectsAvailable}
+                key={item}
+              />
+            </Expandable>
+          );
+        })}
+        {selectedItems.length === 0 && (
+          <div className="text-center">
+            <br />
+            <br />
```

Comment:
I'd avoid `br`. If you need more whitespace, can you use CSS?

We have existing styles for empty state than you should be able to use.